src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/DisplayFormatAttribute.cs

```diff
@@ -45,5 +69,45 @@ public DisplayFormatAttribute()
         ///     Gets or sets a value indicating whether the field should be html encoded
         /// </summary>
         public bool HtmlEncode { get; set; }
+
+		/// <summary>
+		///     Gets or sets the <see cref="System.Type" /> that contains the resources for <see cref="NullDisplayText" />.
```

Comment:
nit: don't need `System.` here or in the `InvalidOperationException` ref near the bottom of the file